strategies/panoptic_getcollateral_example.py

```diff
@@ -0,0 +1,269 @@
+import asyncio
```

Comment:
I suspect this file should be renamed to something like `panoptic_singleleg_option_example.py`